libs/datamodel/core/tests/renderer/simplification.rs

```diff
@@ -52,3 +53,22 @@ fn test_render_relation_name_on_self_relations() {
 
     assert_eq!(rendered, input);
 }
+
+#[test]
+fn experimental_features_roundtrip() {
```

Comment:
I think a better place for this test would be `parser_renderer_dml` where we do exactly those kind of roundtrips already. Just realized that this file is just confusingly placed in the `dmmf` module and this is why you probably didn't find it.